src/accsr/remote_storage.py

```diff
@@ -512,7 +525,12 @@ def driver(self) -> StorageDriver:
         return storage_driver_factory(**self.driver_kwargs)
 
     def _execute_sync(
-        self, sync_object: SyncObject, direction: Literal["push", "pull"], force=False
+        self,
+        sync_object: SyncObject,
+        file_count: Tuple[int, int],
+        direction: Literal["push", "pull"],
+        force=False,
+        use_pbar: Optional[bool] = None,
```

Comment:
docstring missing. Will this override `self.use_pbar`?

Reply:
Added